carbonmark/components/pages/Project/Retire/Pool/RetireInputs.tsx

```diff
@@ -332,7 +328,11 @@ export const RetireInputs: FC<Props> = (props) => {
               message: t`Could not calculate Total Cost`,
             },
             validate: {
-              moreThanZero: (v) => Number(v) > 0 || t`Total Cost is required`,
+              //Add form level validation for totalPrice exceeding the fiat max
+              exceedFiatMax: (v) =>
+                !(
+                  paymentMethod === "fiat" && Number(props.fiatMax) < Number(v)
+                ) || getValidations().totalPrice.max.message,
```

Comment:
I think this is what's causing the double error that jabby noted in his screenshot.